hibike/README.md

```diff
@@ -339,7 +341,35 @@ Note: These assignments are also fairly random and may not all even be
     Direction:
     BBB <-- SD
 
-7. Error Packet: Sent to indicate an error occured. 
+7. Heart Beat Request: BBB requests SD message for connectivity purposes.
+    - This message pathway is a two way street, both BBB and SD can send requests and send responses to the other
+    - Payload is currently unused, but can be used for future functionality in keeping track of individual heartbeat requests and responses (for latency purposes)
+
+    Payload format:
+
+        +---------------+
+        |       ID      |
+        |    (8 bits)   |
+        +---------------+
+
+    Direction:
+    BBB --> SD
+
+8. Heart Beat Response: SD sends BBB message for enumeration purposes.
```

Comment:
Similar comment here.

Also, please explain a little about how heartbeats work.  (I.E. a heart beat requests results in a heart beat response)